en_ID/pages/faq.md

```diff
@@ -210,6 +210,72 @@ Tip: If clip isn't working, you can locate the hidden .ssh folder, open the file
 
 * If prompted, confirm your GitHub password.
 
+## Add new SSH key for Linux
+
+To configure your GitHub account to use your new (or existing) SSH key, you'll also need to add it to your GitHub account.
+
+### Generating a new SSH key
+
+1. Open Terminal.
+2. Paste the text below, substituting in your GitHub email address.
+
+```
+$ ssh-keygen -t ed25519 -C "your_email@example.com"
```

Comment:
why choosing `ed25519` instead of RSA?

Reply:
Because Ed25519 offers better security, for better long term, ED25519 already encrypts keys to the more secure OpenSSH format, and a lot of people recommend using Ed25519 instead of RSA.